services/transaction_body.proto

```diff
@@ -142,6 +142,11 @@ message TransactionBody {
      */
     ContractDeleteTransactionBody contractDeleteInstance = 22;
 
+    /**
+     * An Ethereum encoded transaction.
+     */
+    EthereumTransactionBody ethereumTransaction = 50;
```

Comment:
Curious why it was moved here out of alphabetical order.

Reply:
To put it next to contract functions.  My read is that this list is functional order.